src/components/Registration/Manual/SearchAttendee.vue

```diff
@@ -13,13 +13,11 @@ const emit = defineEmits(['print'])
 const menuOpen = ref(false)
 const query = ref('')
 
-watch(query, (value) => {
-  if (value.includes('@')) {
-    searchAttendeeStore.getAttendee('', value) // is an email
-  } else if (value == '') {
-    searchAttendeeStore.clearAttendees()
+watch(query, (newValue) => {
+  if (newValue === '') {
+    setTimeout(() => searchAttendeeStore.clearAttendees(), 700)
```

Comment:
this is to cclear the array? why not just clear on the next search

Reply:
because it will keep clear on every input and looks very distracting